components/cosmetic_filters/renderer/DEPS

```diff
@@ -1,4 +1,5 @@
 include_rules = [
+  "+brave/common",
```

Comment:
this isn't allowed, it's a layering violation. Instead of passing in `GetDynamicParamsCallback` you can wrap it in a closure that just returns the value of de_amp_enabled